packages/material-ui/src/Typography/Typography.test.js

```diff
@@ -1,25 +1,38 @@
 import React from 'react';
 import { assert } from 'chai';
-import { createShallow, getClasses } from '@material-ui/core/test-utils';
+import {
+  createShallow,
+  createMount,
+  describeConformance,
+  getClasses,
+} from '@material-ui/core/test-utils';
 import Typography from './Typography';
 
 describe('<Typography />', () => {
+  let mount;
   let shallow;
   let classes;
 
   before(() => {
+    mount = createMount();
     shallow = createShallow({ dive: true });
     classes = getClasses(<Typography />);
+
+    describeConformance(<Typography />, {
```

Comment:
React element vs component. What prevents us from using a component? Is this done in order to add the required properties a component might need?

Reply:
That was the primary concern, yes. It just came down to two arguments vs. one argument

I don't really care about `Component` + `props` => `<Component {...props} />` vs `<Component {...props} />` => `cloneElement`. I never found a comprehensive comparison of these two approaches.

Reply:
Yeah ok, I was curious, it's fine by me. Even our component API mixes a bit the two. Basically, we use an element API as long as we don't need to use cloneElement.